js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -19,6 +22,11 @@ action=Aktion
 join=Beitreten
 events=Ereignisse
 sessions=Sessions
+searchForSession=Sitzung suchen
+signOutAllActiveSessions=Alle aktiven Sitzungen abmelden
+revocation=Widerrufen
+sessionsType.allSessions=Alle Sitzungsarten
+sessionExplain=Sitzungen sind Sitzungen von Benutzern in diesem Bereich und die Clients, auf die sie innerhalb der Sitzung zugreifen.
```

Comment:
Sitzungen sind Sitzungen von Benutzern in diesem Bereich sowie den Clients, auf die sie innerhalb der Sitzung zugreifen

Reply:
Jap that slipped through, when I realized that I kept the term "Realm". Will change the line above to:
"Sitzungen von Benutzern in diesem Realm und die Clients, die innerhalb der Sitzung verwendet werden."